### DISCUSSION TITLE
is there a way to change/update vectors in index?

### DISCUSSION THREAD

**Discussion by hongyuanmei** [facebookresearch/faiss#3643] [General]
Hi FAISS community, 

I am considering using faiss in a setting that I dynamically update the vectors added into the index.  
E.g., I may want
```
import faiss                   # make faiss available
index = faiss.IndexFlatL2(d)   # build the index
index.add(xb)                  # add vectors to the index
index.update(ids, new_vecs)    # a hypothetical method that updates the vectors of ids to new_vecs
```
Is there a way to do this in faiss? 

Thanks very much in advance! 

Best

**Comment by mdouze** [comment]
For brute-force search you could be interested in 
https://github.com/facebookresearch/faiss/wiki/Brute-force-search-without-an-index
